doc/source/workflows/metadata.rst

```diff
@@ -120,6 +120,25 @@ And you can also do this in a nested way:
     assert workflow.get_metadata("counter", "incr_3")["user_metadata"] == {"current_n": 3}
     assert workflow.get_metadata("counter", "incr_4")["user_metadata"] == {"current_n": 4}
 
+Available Metrics
+-----------------
+**workflow level**
+
+- status: workflow states, can be one of RUNNING, FAILED, RESUMABLE, CANCELED or SUCCESSFUL.
+- user_metadata: a python dictionary of user input metadata that is passed in ``workflow.run()``.
+- stats: workflow running stats, including workflow start time and end time.
+
+**step level**
+
+- name:
+- workflow:
+- workflow_refs:
+- step_options:
+- user_metadata: a python dictionary of user input metadata that is passed in ``step.options()`` or ``@workflow.step()``.
+decorators
+- stats: step running stats, including step start time and end time.
+
+
```

Comment:
@iycheng can you help with the four missing definitions in the step level? those are retrieved from existing inputs.json. I know what those are but you might have better wording than me on the one-line definitions of each.